### PR TITLE
AC-533 Adding IntroSlider for first time users

### PR DESCRIPTION
**Pull request by Asgardian8740**
AC-533 Adding IntroSlider for first time users

## Description of what I changed
Added Intro-Slider for the first Time user. 

Screenshot_20190321-153806
Screenshot_20190321-153810
Screenshot_20190321-153814
Screenshot_20190321-153918
Screenshot_20190321-153923
Screenshot_20190321-153934

Changes have been made to the UI of LoginFrament.xml also.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-533

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-  ~ compliance with the License. You may obtain a copy of the License at
-  ~ http://license.openmrs.org
-  ~
-  ~ Software distributed under the License is distributed on an "AS IS"
-  ~ basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
-  ~ License for the specific language governing rights and limitations
-  ~ under the License.
-  ~
-  ~ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
--->
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/AndroidManifest.xml]: @Asgardian8740 
could you add back the OpenMRS license at line 1?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardActivity.java [hunk 2]
```diff
@@ -46,6 +48,8 @@ protected void onCreate(Bundle savedInstanceState) {
             // Pre-Marshmallow
         }
         */
+
+
         super.onCreate(savedInstanceState);
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardActivity.java]: @Asgardian8740 Don't add Empty lines unnecessarily. Try making changes to as many fewer files as Possible, so that there would be less merge Conflicts.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    public void setFirstTimeLaunch(boolean isFirstTime) {
+        editor.putBoolean(IS_FIRST_TIME_LAUNCH, isFirstTime);
+        editor.commit();
+    }
+
+    public boolean isFirstTimeLaunch() {
+        return pref.getBoolean(IS_FIRST_TIME_LAUNCH, true);
+    }
+}
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java]: @Asgardian8740 you can use the `getOpenMRSSharedPreferences` method of OpenMRS instance to get the sharedPreferences. you don't need to have a file for it.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    SharedPreferences pref;
+    SharedPreferences.Editor editor;
+    Context _context;
+
+    // shared pref mode
+    int PRIVATE_MODE = 0;
+
+    private static final String PREF_NAME = "welcome";
+    private static final String IS_FIRST_TIME_LAUNCH = "IsFirstTimeLaunch";
+
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java]: you can have added constants to ApplicationsConstants file in Utilities folder.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java [hunk 5]
```diff
@@ -250,12 +248,10 @@ private void initViewFields(View root) {
...
     public void onResume() {
         super.onResume();
-
         final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(OpenMRS.getInstance());
         boolean syncState = prefs.getBoolean("sync", true);
         setSyncButtonState(syncState);
         hideUrlLoadingAnimation();
-
         bindDrawableResources();
     }
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java]: Please don't make changes to files which are not related to the Issue. This would make other PR have merge conflicts.

**Comment by VANKINEENITAWRUN** [comment]
@Asgardian8740 Thanks for your PR.

File: openmrs-client/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Asgardian8740) [reply openmrs-client/src/main/AndroidManifest.xml]: Oh thank you.......I forgot to add.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardActivity.java (hunk 2 above)
Comment (Asgardian8740) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardActivity.java]: Thanks

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java (hunk 3 above)
Comment (Asgardian8740) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/introslider/PreferenceManager.java]: Purpose for having separate Preference Manger for intro-slider is to make life easier for the future developer working on this project
